source/Azure.Common/Sashimi.Azure.Common.csproj

```diff
@@ -8,7 +8,8 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <Nullable>enable</Nullable>
     <PackageProjectUrl>https://github.com/OctopusDeploy/Sashimi/</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/OctopusDeploy/Sashimi/blob/master/LICENSE.txt</PackageLicenseUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <LangVersion>default</LangVersion>
```

Comment:
I guess `PackageLicenseExpression` is the new thing?

Why do we have `LangVersion` for this file and not for `Sashimi.Aws.Common.csproj`

Reply:
> I guess `PackageLicenseExpression` is the new thing?

Yes, it was complaining about it when we build with cake

> Why do we have LangVersion for this file and not for Sashimi.Aws.Common.csproj

Good point, I'm going to start a discussion in modularity slack to see where we want to go